dashboard_service/settings/common.py

```diff
@@ -164,10 +164,8 @@
         "client_secret": AUTH0_CLIENT_SECRET,
         "client_kwargs": {
             "scope": "openid profile email",
-            "prompt": "login",  # forces login prompt
         },
         "server_metadata_url": f"https://{AUTH0_DOMAIN}/.well-known/openid-configuration",
-        "authorize_params": {"isPasswordlessFlow": True},  # required to trigger passwordless login
```

Comment:
This was only used for some custom login in the "Classic Universal Login Page", so can now be removed